libraries/APM_Control/AR_AttitudeControl.cpp

```diff
@@ -601,7 +601,7 @@ bool AR_AttitudeControl::get_forward_speed(float &speed) const
     if (!_ahrs.get_velocity_NED(velocity)) {
         // use less accurate GPS, assuming entire length is along forward/back axis of vehicle
         if (AP::gps().status() >= AP_GPS::GPS_OK_FIX_3D) {
-            if (fabsf(wrap_180_cd(_ahrs.yaw_sensor - AP::gps().ground_course_cd())) <= 9000) {
+            if (abs(wrap_180_cd(_ahrs.yaw_sensor - AP::gps().ground_course_cd())) <= 9000) {
```

Comment:
Technically this whole thing would actually be better as:
``` c++
fabsf(wrap_180(degrees(_ahrs.yaw_sensor) - AP::gps().ground_course())) <= 90)
```

(the GPS ground_course_cd() function is doing a float mult + conversion, whereas if you operate purely in the float range you only pay the cost of one float mult)

Reply:
Don't think that change is quite right (mixing cd and degrees).  And it might be less maths, but not my fight on this PR :-)

Reply:
Sorry, fixed it, ahrs.yaw is the angle in radians as a float. Anyways, thats not mixing units :)